src/renderer/components/FormInputs/Toggle.tsx

```diff
@@ -10,15 +11,25 @@ type ToggleProps = {
   value: boolean;
   onChange: (val: boolean) => void;
   disabled?: boolean;
+  tooltipTitle?: string;
 };
 
-export const Toggle = ({ label, description, value, onChange, disabled }: ToggleProps) => {
+export const Toggle = ({ label, description, value, onChange, disabled, tooltipTitle }: ToggleProps) => {
   return (
     <div>
       <FormControlLabel
         labelPlacement="start"
         control={
-          <Switch checked={value} onChange={(e) => onChange(e.target.checked)} color="primary" disabled={disabled} />
+          <Tooltip title={tooltipTitle}>
+            <div>
```

Comment:
Do we need this `div`? I assume you added it for a reason.

Reply:
yeah the Tooltip won't work if it is wrapping a disabled component so you need a div in between